CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ endif()
 if(ENABLE_DATASOURCE)
   list(APPEND root_components_needed ROOTDataFrame)
 endif()
-find_package(ROOT REQUIRED COMPONENTS ${root_components_needed})
+find_package(ROOT 6.28.04 REQUIRED COMPONENTS ${root_components_needed})
 if((ENABLE_RNTUPLE) AND (${ROOT_VERSION} VERSION_LESS 6.28.02))
     message(FATAL_ERROR "You are trying to build podio with support for the new ROOT NTuple format, but your ROOT version is too old. Please update ROOT to at least version 6.28.02")
 endif()
```

Comment:
Is this `if` still needed since greater version is now required?

I guess the README should also get updated as currently it states `ROOT 6.08.06` :slightly_smiling_face:

Reply:
Removed